### PR TITLE
 #166790067: Reporting an article

### PR DESCRIPTION
**Pull request by kagabof**
#### What does this PR do?

- Enables users to report articles.

#### Description of Task to be completed?

- Add actions, reducers and initial state for reporting an article.
- Add a model for reporting an article.
- Add tests for reporting articles.

#### How should this be manually tested?

- By using reporting an article.

#### What are the relevant pivotal tracker stories?

[#166790067](https://www.pivotaltracker.com/story/show/166790067)

#### Screenshots

image

### CODE REVIEW

**Review by shaluchandwani — CHANGES_REQUESTED** [review]
Good Work on the PR @kagabof. I would be great if you will remove the border for the select buttons and text in your report article page. Also if the R is capital in report article, it will look good.

**Comment by kagabof** [comment]
@shaluchandwani, well do. just add a message that indicates that you have no bookmarked articles.

**Review by shaluchandwani — APPROVED** [review]
Approved!

**Review by pngabo — APPROVED** [review]

File: src/assets/scss/components/articlesDownSide.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            margin-right: 40px;
+        }
+        &_concel{
+            background: #F9FCFB;
+            border: 1px solid #B5B5B5;
+            color: #C81010;
+        }
+        
+    }
+}
```
Comment (pngabo) [inline src/assets/scss/components/articlesDownSide.scss]: add a line at the end of the file

**Review by Joe-Joseph — APPROVED** [review]

**Comment by billkabanga** [comment]
Great work on this PR, please have the content of this modal centre-aligned
Screenshot 2019-09-17 at 10 09 31

Also fix this, I think it should be `harassment` not `harsment`
Screenshot 2019-09-17 at 10 07 41

**Pull request merged**